cv_next/app/cv/[cvId]/components/commentSection/comment.tsx

```diff
@@ -18,6 +18,7 @@ import { setResolved } from "@/app/actions/comments/setResolved";
 import { deleteComment } from "@/app/actions/comments/deleteComment";
 import Alert from "../../../../../components/ui/alert";
 import Tooltip from "../../../../../components/ui/tooltip";
+//what the ^
```

Comment:
Fix these

Reply:
If the `what the` refers to the relative import hell, you can just fix it with `@`s.

Reply:
done